app/api/og/route.tsx

```diff
@@ -1,24 +1,82 @@
+              fill="black"
+            ></path>
+          </svg>
+        </div>
+        <div
+          style={{
+            display: "flex",
+            fontSize: 40,
+            fontStyle: "normal",
+            color: "black",
+            marginTop: 30,
+            lineHeight: 1.8,
+            whiteSpace: "pre-wrap",
+          }}
+        >
+          <b>{siteName}</b>
```

Comment:
```suggestion
            color: "black",
            marginTop: 30,
            lineHeight: 1.8,
            whiteSpace: "pre-wrap",
          }}
        >
          <strong>{siteName}</strong>
```
从语义上应该用 `<figure />` 系列标签。

Reply:
这个是用来生成图片的，不支持这个标签
https://github.com/vercel/satori/blob/main/src/handler/presets.ts